openc3/lib/openc3/api/limits_api.rb

```diff
@@ -364,5 +364,35 @@ def _limits_group(group_name, action:, scope:, token:)
         TargetModel.set_packet(last_target_name, last_packet_name, packet, scope: scope)
       end
     end
+
+    # Gets an item. The code below is mostly duplicated from tlm_process_args in tlm_api.rb.
+    #
+    # @param target_name [String] target name
+    # @param packet_name [String] packet name
+    # @param item_name [String] item name
+    # @param scope [String] scope
+    # @return Hash The requested item based on the packet name
+    def _get_item(target_name, packet_name, item_name, scope:)
+      requested_item = nil
+      if packet_name == 'LATEST'
+        latest = -1
+        TargetModel.packets(target_name, scope: scope).each do |packet|
+          item = packet['items'].find { |item| item['name'] == item_name }
+          if item
+            # TODO: Fixme: This should be using the CVT not topics - Will possibly choose wrong packet if mixed with stored
```

Comment:
Feels like a good opportunity to fix this?

Reply:
I didn't write that comment and I'm not sure what it means. The topic returns the time which we use to find 'latest'. The CvtModel only returns values.

Reply:
The CVT stores the most recent of each packet.  The problem with topics is that they contain both realtime and stored data.  Whenever you use Topics you have to make sure the data you use isn't stored, and we aren't doing that here.